website/docs/using-unleash/troubleshooting/email-service.md

```diff
@@ -0,0 +1,35 @@
+---
+title: The email service is not working correctly on my self-hosted Unleash instance
+---
+
+When setting up your self-hosted Unleash instance, one of the available options is to configure an [email service](https://docs.getunleash.io/using-unleash/deploy/email-service) that will allow Unleash to send reset password and welcome emails to users.
+
+Here's how to troubleshoot some common issues related to the email service.
+
+## Configuration
+
+The most common issues arise from misconfiguration. Please take into account the following documentation:
```

Comment:
Maybe "Please refer to the following documentation for guidance"

Reply:
Addressed in https://github.com/Unleash/unleash/pull/5333/commits/ba1fc8a0bc587aa2a1ed4a70c3c5777ec0106ef4